tm2/pkg/crypto/keys/client/addpkg.go

```diff
@@ -195,7 +195,7 @@ func signAndBroadcast(
 
 	signedTx, err := SignHandler(sopts)
 	if err != nil {
-		return errors.Wrap(err, "sign tx")
+		return fmt.Errorf("unable to sign tx: %w", err)
```

Comment:
It looks strange that we need to do this. Can’t we improve the display logic of the errors package?